src/js/containers/covid19/homepage/CovidFeatureContainer.jsx

```diff
@@ -135,7 +136,7 @@ const CovidFeatureContainer = ({
                     </div>
                     <div className="homepage-feature-description adv-search-spending-profile__text">
                         <p>Our newest profile page shows you official COVID-19 spending information as submitted by federal agencies. Learn more about <strong className="homepage-feature-description_weight_bold">who received funding, which agencies outlayed funds,</strong> and <strong className="homepage-feature-description_weight_bold">which programs were funded</strong>.</p>
-                        <p>All COVID-19 spending data is <strong className="homepage-feature-description_weight_bold">available for download</strong> on the profile page with one click.</p>
+                        <p>All COVID-19 spending data is <strong className="homepage-feature-description_weight_bold">available for download</strong> on the profile page with one click. You can also read about our datasets and calculations on the <a href="#/disaster/covid-19/data-sources">Data Sources &amp; Methodology</a> page </p>
```

Comment:
Andrew's doc says "Add the following sentence as a third paragraph" - this may have changed after discussing with him but wanted to make sure!

Reply:
Thanks for catching this - worked with Andrew on this, and we decided to keep this text to 2 paragraphs.